integration-test/helpers/harness.js

```diff
@@ -12,10 +12,17 @@ const DATA_DIR_PREFIX = 'ddg-temp-'
  * @param {string} domain
  */
 export async function setupMockedDomain (page, domain) {
+    const contentType = {
+        html: 'text/html',
+        css: 'text/css',
+        default: 'text/plain'
+    }
     await page.route(`${domain}/**/*`, (route, request) => {
         const { pathname } = new URL(request.url())
+        const fileType = pathname.split('.').pop()
         return route.fulfill({
             status: 200,
+            contentType: contentType[fileType] || contentType.default,
```

Comment:
Mhm, why is this needed? Sorry, I forgot.

Reply:
@GioSensation I think this is the first time we've used `setupMockedDomain` on Playwright's Safari engine, but on mocked tests the CSS file wasn't loading because the content type wasn't set. This now sets up the correct content type for the mocked response depending on the file extension pulled from the URL.